### DISCUSSION TITLE
Combine simulations from different .pksim files created from MoBi?

### DISCUSSION THREAD

**Discussion by jhunt8** [Open-Systems-Pharmacology/Forum#1345] [Questions & Problems] [answered]
Hi All, I have two PK-Sim population simulations in separate .pksim files that were both created from MoBi using the "send to PK-Sim for population simulation" feature. I would like to combine both of these simulations into one .pksim file to use "Compare Results", etc. 

It seems that PK-Sim simulations sent from MoBi in this way can't be saved as .pkml files and thus can't be imported as described: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/importing-exporting-project-data-models. Is there another way to combine these files?

**Comment by tobiasK2001** [comment, marked as answer]
Hi,

there is some easy way to Combine these Simulation in one PK-Sim file:
Just create or select an existing Working journal for the first created PK-Sim file. Than you attach the Population simulation by right clicking on it and select "Add to journal" from the sub-menu.
![image](https://user-images.githubusercontent.com/25585248/196620774-a5453702-563b-44b2-b690-b955d8b6801f.png)
 Than you open the WJ from the first file in second PK-Sim file. Go to the journal Page in the Journal view/related items and click the reload simulation button 
![image](https://user-images.githubusercontent.com/25585248/196621106-2f04bbfe-05de-4236-8c68-689eb9ff4ecd.png)
next to the Attached Simulation.
This loads now your first PK-Sim Simulation in your second PK-Sim file were you can compare both.
N.B. a lot more can be interchanged, attached and reloaded between PK-Sim and MoBi files via the Working Journal. Check this out.
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/working-journal#project-documentation
Hope this might help you 
Tobias

**Reply by jhunt8** [reply]
Very useful, thank you.
